docs/user/installation.rst

```diff
@@ -98,36 +105,7 @@ pip:
 Extra Steps for Garage Developers
 ---------------------------------
 
-If you plan on developing the garage repository, as opposed to simply using it as a library, you will probably prefer to install your copy of the garage repository as an editable library instead. After installing the pre-requisites using the instructions in `Install Environment Dependencies (Optional)`_, you should install garage in your environment as below.
-If you would like to contribute changes back to garage, please also read :code:`CONTRIBUTING.md`.
-
-- pipenv
-
-.. code-block:: bash
-
-    cd path/to/garage/repo
-    pipenv --three
-    pipenv install --pre -e '.[all,dev]'
-
-
-- conda
-
-.. code-block:: bash
-
-    conda activate myenv
-    pip uninstall garage  # To ensure no existing install gets in the way.
-    cd path/to/garage/repo
-    pip install -e '.[all,dev]'
-
-
-- virtualenv
-
-.. code-block:: bash
-
-    source myenv/bin/activate
-    pip uninstall garage  # To ensure no existing install gets in the way.
-    cd path/to/garage/repo
-    pip install -e '.[all,dev]'
+See `here <setting_up_your_development_environment.md#installing-garage-as-an-editable-package>`_
```

Comment:
This seems not working in the [latest doc](https://garage.readthedocs.io/en/latest/user/installation.html).

Reply:
@yeukfu can you file an issue with more details?

Reply:
Sure.